test/start-stop.node.js

```diff
@@ -212,6 +211,9 @@ types.forEach((type) => {
     })
 
     describe('start and stop multiple times', () => {
+      // TODO: wont work on windows until we get /shutdown implemented in js-ipfs
```

Comment:
It's now working, see PR https://github.com/ipfs/js-ipfsd-ctl/pull/205

Reply:
Nice... is it ready to review and merge?

Reply:
waiting for a review by @diasdavid 

My PR has a link to js-ipfs/master (not using npm) so it can be used immediately.